.travis.yml

```diff
@@ -39,20 +39,20 @@ env:
     - OS_CODE_NAME2=bionic
     - ARCH=amd64
     - ROS_BUILDFARM_PULL_REQUEST_BRANCH=$TRAVIS_PULL_REQUEST_BRANCH
+os: linux
 dist: xenial
-matrix:
+language: python
```

Comment:
With this the `language` key can probably be removed from each jobs section.

Reply:
7fd7982